build.mjs

```diff
@@ -75,32 +75,208 @@ const ES_BUILD_OPTIONS = {
+  ...FULL({ "client-amplify": ["Amplify", ["amplify"]] }),
+  ...FULL({ "client-appsync": ["AppSync", ["appsync"]] }),
+  ...FULL({ "client-location": ["Location", ["geo"]] }),
+};
+const _GameTech = {};
+const _InternetOfThings = {};
+const _MachineLearningAndArtificialIntelligence = {
+  ...FULL({ "client-bedrock": ["Bedrock", ["bedrock"]] }),
+  ...FULL({ "client-bedrock-agent": ["BedrockAgent", ["bedrock-agent"]] }),
+  ...FULL({
+    "client-bedrock-runtime": ["BedrockRuntime", ["bedrock-runtime"]],
+  }),
+  ...FULL({
+    "client-bedrock-agent-runtime": [
+      "BedrockAgentRuntime",
+      ["bedrock-agent-runtime"],
+    ],
+  }),
+  ...FULL({ "client-polly": ["Polly", ["polly"]] }),
+  ...FULL({ "client-rekognition": ["Rekognition", ["rekognition"]] }),
+  ...FULL({ "client-textract": ["Textract", ["textract"]] }),
+  ...FULL({ "client-translate": ["Translate", ["translate"]] }),
+};
+const _ManagementAndGovernance = {
+  ...FULL({ "client-appconfig": ["AppConfig", ["appconfig"]] }),
+  ...FULL({ "client-appconfigdata": ["AppConfigData", ["appconfigdata"]] }),
+  ...FULL({ "client-cloudformation": ["CloudFormation", ["cloudformation"]] }),
+  ...FULL({ "client-cloudwatch": ["CloudWatch", ["monitoring"]] }),
+  ...STD({ "client-cloudwatch-logs": ["CloudWatchLogs", ["logs"]] }),
+  ...STD({ "client-cloudwatch-events": ["CloudWatchEvents", ["events"]] }),
+  ...FULL({ "client-service-catalog": ["ServiceCatalog", ["servicecatalog"]] }),
+  ...STD({ "client-ssm": ["SSM", ["ssm"]] }),
+};
+const _Media = {
+  ...FULL({ "client-mediaconvert": ["MediaConvert", ["mediaconvert"]] }),
+};
+const _MigrationAndTransfer = {};
+const _NetworkingAndContentDelivery = {
+  ...FULL({ "client-api-gateway": ["APIGateway", ["apigateway"]] }),
+  ...FULL({ "client-apigatewayv2": ["ApiGatewayV2", ["apigateway"]] }),
+  ...FULL({
+    "client-elastic-load-balancing-v2": [
+      "ElasticLoadBalancingV2",
+      ["elasticloadbalancing"],
+    ],
+  }),
+};
+const _QuantumTechnologies = {};
+const _Robotics = {};
+const _Satellite = {};
+const _SecurityIdentityAndCompliance = {
+  ...FULL({ "client-acm": ["ACM", ["acm"]] }),
+  ...STD({
+    "client-cognito-identity": ["CognitoIdentity", ["cognito-identity"]],
+  }),
+  ...STD({
+    "client-cognito-identity-provider": [
+      "CognitoIdentityProvider",
+      ["cognito-idp"],
+    ],
+  }),
+  ...FULL({ "client-iam": ["IAM", ["iam"]] }),
+  ...STD({ "client-kms": ["KMS", ["kms"]] }),
+  ...STD({ "client-secrets-manager": ["SecretsManager", ["secretsmanager"]] }),
+  ...FULL({ "client-sso": ["SSO", ["sso", "identitystore"]] }),
+  ...FULL({ "client-sso-admin": ["SSOAdmin", ["sso", "identitystore"]] }),
+  ...FULL({ "client-sso-oidc": ["SSOOIDC", ["sso", "identitystore"]] }),
+  ...STD({ "client-sts": ["STS", ["sts"]] }),
+};
+const _Storage = {
+  ...FULL({ "client-efs": ["EFS", ["elasticfilesystem"]] }),
+  ...STD({ "client-s3": ["S3", ["s3"]] }),
+  ...STD({ "lib-storage": ["Upload", ["s3"]] }),
+};
+
```

Comment:
This seems overly complex. Since we have a hierarchical structure i would structure the data differently.
We can keep the original `SDK_DATA` and set an optional boolean to indicate if it's `full` or not `std`. We only need to distinguish included SDKs between `full` and `std` since `no-sdk` contains none of these:
```javascript
//format: {CLIENT_IMPORT_NAME}: [{SDK_NAME},{ENDPOINT_NAME},{FULL_SDK_ONLY}]
const SDK_DATA = {
  "client-eventbridge": ["EventBridge", "events", true],
  "client-dynamodb": ["DynamoDB", "dynamodb"],
  ,...
  }
```

We can also distinguish them via comments rather than separate variables:
```javascript
// Application integration
...
//Blockchain
...
//Compute services


```

Reply:
> This seems overly complex. Since we have a hierarchical structure i would structure the data differently.
> We can keep the original `SDK_DATA` and set an optional boolean to indicate if it's `full` or not `std`. We only need to distinguish included SDKs between `full` and `std` since `no-sdk` contains none of these:

You pointed out that by using `cargo --features no-sdk`, we could just use the `full` or `std` binary decision, but I don't think that would work. At this time, we are `using cargo --features lambda` for binary generation for lambda, even in the case of `no-sdk`. This is because we do not want to bundle test code.

<img width="610" alt="image" src="https://github.com/user-attachments/assets/d62917ca-9ef0-45f8-add0-8b7856b03395">

By reviewing the above process, we could exclude the test code in the `no-sdk` case, but this time it would affect the binary for linux/darwin (llrt test would not work)...

Therefore, at the present time, it is desirable to be able to make a three-value judgment here: no | std | full.

> We can also distinguish them via comments rather than separate variables:

Your point here makes sense! I will review it immediately. :)

Reply:
I think I wasn't expressing myself clear :) We have 3 "flavors" when building the JS like you described:
 no | std | full.
 However we only need to distinguish between std & full when building the `SDK_DATA` list. If we have `no` we simply ignore the values `SDK_DATA` in the list.